src/apps/frontend/components/typography/label-medium.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { PropsWithChildren } from 'react';
+
+const LabelMedium: React.FC<PropsWithChildren> = ({
+  children,
+}) => (
+  <p className="text-base font-medium text-black">
```

Comment:
Why are using p for heading and not label tags? This is semantically invalid HTML